developers/contributor-guide/current/index.md

```diff
@@ -3,29 +3,51 @@ layout: layout-documentation
 bodyclass: ["page--guides", " "]
 solution: contributor-guide
 sub-menu: index
-solution-order: 3
 title: Introduction
 intro: This guide provides you with all the resources you need to develop on Weaviate and it's related offerings.
 description: About
 tags: ['index', 'contributor-guide']
 menu-order: 1
 open-graph-type: article
 og-img: documentation.jpg
-toc: false
+toc: true
 redirect_from:
   - /documentation/contributor-guide/current/index.html
   - /documentation/contributor-guide/current/
 ---
 
-## How to contribute
-There's roughly two ways you can contribute to the development of Weaviate: 
-1. By directly contributing to the open source code in Github, or
-2. By using, testing, sharing, giving feedback and proposing ideas via one of our communication channels.
+![img](/img/guides/contributor/Weaviate.png)
 
-This guide contains information on our development philosophy, standards, tips and benchmarks of Weaviate and its tools (e.g. Contextionary and Clients). You can start contributing by for example looking at [good first issues](https://github.com/semi-technologies/weaviate/labels/good-first-issue).
+# Welcome to the Weaviate's Contributing Guide 
 
-But also if you're not here to contribute in code, you help Weaviate improve and grow by testing and giving us feedback through creating [issues](https://github.com/semi-technologies/weaviate/issues), asking questions on [stackoverflow](https://stackoverflow.com/questions/tagged/weaviate) or in the [community Slack channel](https://join.slack.com/t/weaviate/shared_invite/zt-goaoifjr-o8FuVz9b1HLzhlUfyfddhw).
+If you’re using Weaviate or if you like the project, please <a href="https://github.com/semi-technologies/weaviate">⭐</a> this repository to show your support!
 
-## More Resources
+[Weaviate](https://weaviate.io/) is a cloud-native, modular, real-time vector search engine.
 
-{% include docs-support-links.html %}
+## Contributing to Weaviate
+
+🎉 First off, thank you for taking the time to contribute! 🎉
+
+We are delighted to have you here. We are thrilled that you want to contribute to the Weaviate core, as together we can make Weaviate even better. We strive to build an engaging community and we encourage you to participate, share your ideas and make friends. As you become more acquainted with the community, there are several initiatives to consider, which are listed below. Everyone is welcome to join our [Slack](https://weaviate.slack.com/) channel and interact with other members of the community.
```

Comment:
`As you become more acquainted with the community, there are several initiatives to consider,`

->

`Listed below are several ways to get involved with the community`